python/jetToolbox_cff.py

```diff
@@ -700,6 +713,10 @@ def jetToolbox( proc, jetType, jetSequence, outputFile,
 	elemToKeep += [ 'drop *_selectedPatJets'+jetALGO+'PF'+PUMethod+'_calo*_*' ]
 	elemToKeep += [ 'drop *_selectedPatJets'+jetALGO+'PF'+PUMethod+'_tagInfos_*' ]
 
+	if runOnData:
+		from PhysicsTools.PatAlgos.tools.coreTools import removeMCMatching
+		removeMCMatching(proc, names=['Jets'], outputModules=[])
```

Comment:
Actually I take it back.. I think these three lines should be at the very end of the script, once the module outputFIle is created.